### PR TITLE
Local page

### PR DESCRIPTION
**Pull request by luckyagarwal**
Added Bottom Sheet for Local Tab:
| | |
|:-------------------------:|:-------------------------:|
|Image 1 Dark  |  Image 1.2 Dark 
| Image 1 Light | Image 1,2 Light |

### CODE REVIEW

File: SwiftLeeds/Views/Components/BottomSheetView.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    internal init (isOpen: Binding<Bool>,
+                   maxHeight: CGFloat){
+        self.minHeight = maxHeight * Constants.minHeightRatio
+        self.maxHeight = maxHeight
+        self._isOpen = isOpen
+    }
+    
+    var body: some View {
+        GeometryReader{ geometry in
+            VStack{
```
Comment (pdamonkey) [inline SwiftLeeds/Views/Components/BottomSheetView.swift:30]: Very minor, just add a space before the {

File: SwiftLeeds/Views/Components/BottomSheetView.swift [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                VStack(spacing: Padding.cellGap) {
+                    Spacer()
+                    SectionHeader(title: "Local",
+                                  fontStyle: .title2.weight(.semibold),
+                                  foregroundColor: .primary)
+                    ScrollView {
+                        LocalCell(label: "Food",
+                                  imageName:"takeoutbag.and.cup.and.straw.fill",
```
Comment (pdamonkey) [inline SwiftLeeds/Views/Components/BottomSheetView.swift:38]: Minor, add a space after the colon

**Review by SwiftyAlex — COMMENTED** [review]
Great work! 

Just one piece of design feedback - we should look to fix this, but it doesn't necessarily have to be here 😄 

| Example | Issue |
| - | - | 
| image| The images are all aligned to the right. This is a perfectly fine approach, but it means that the images are not optically aligned with each other. We should aim to switch these so that the images are centred within their frames, and the outer frame is aligned to the trailing edge. This would mean the centre point of each image is in-line with the centre of the others. It also looks like some of these images are out of aspect ratio - to get around this, we can set their explicit frame, and then give them as aspect ratio of fit, so they'll all have at most one axis that fills the whole space. |

Happy to give code guidance on the above to whoever needs it, if they need it 😄

File: SwiftLeeds/Views/Components/BottomSheetView.swift [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                                  labelFontStyle: .body)
+                        LocalCell(label: "Best of Leeds",
+                                  imageName: "mappin",
+                                  labelFontStyle: .body)
+                    }
+                }
+                .padding(Padding.screen)
+            }
+            .frame(width: geometry.size.width, height: self.maxHeight, alignment: .top)
+            .background(Color(.secondarySystemBackground))
```
Comment (pdamonkey) [inline SwiftLeeds/Views/Components/BottomSheetView.swift:54]: Should this be `Color.background` to match the My Conference view?

**Review by pdamonkey — APPROVED** [review]

**Pull request merged**